webapp/src/components/github_label_selector/github_label_selector.jsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import BackendSelector from 'components/backend_selector';
+
+export default class GithubLabelSelector extends Component {
```

Comment:
Instead of implementing `shouldComponentUpdate`, we can use React's built-in `PureComponent`

```suggestion
export default class GithubLabelSelector extends React.PureComponent {
```